src/main/java/org/apache/commons/collections4/bloomfilter/LayeredBloomFilter.java

```diff
@@ -89,19 +90,35 @@ public boolean test(BloomFilter x) {
             return true;
         }
     }
+
+    /**
+     * Creates a fixed size layered bloom filter that adds new filters to the list,
+     * but never merges them. List will never exceed maxDepth. As additional filters
+     * are added earlier filters are removed.  Uses SimpleBloomFilters.
+     *
+     * @param shape    The shape for the enclosed Bloom filters.
+     * @param maxDepth The maximum depth of layers.
+     * @return An empty layered Bloom filter of the specified shape and depth.
+     */
+    public static  LayeredBloomFilter<BloomFilter> fixed(final Shape shape, int maxDepth) {
```

Comment:
Q. Why are we adding this factory method? It seems strange to have one factory method creating layers to a fixed depth with no merges when there are countless other combinations that can be used to create the layers. Is this the most common usage of a layered Bloom filter, e.g. described in Zhiwang et al?

Reply:
Do we foresee the need to add more constructors in the future? I wonder if we should consider having a private constructor and only factory methods, or a builder.